client/components/RegionList.js

```diff
@@ -1,9 +1,49 @@
-import React from 'react'
+import React, { useState, useEffect } from "react"
+import fetchData from "../functions/fetchData.js"
+
+const RegionList = (props) => {
+  const [regions, setRegions] = useState([])
+  const [selectedRegion, setSelectedRegion] = useState(null)
+
+  useEffect(() => {
+    fetchRegions
+  }, [])
+
+  const fetchRegions = async (id) => {

```

Comment:
Where does this `id` value come from? I see on line 9 you are using this function, but you aren't passing it any arguments